src/modules/include/moduleWrapper.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <functional>
+#include <string>
+#include "configuration.hpp"
+
+using namespace std;
+
+struct ModuleWrapper {
+    function<void()> start;
+    function<int(const Configuration&)> setup;
+    function<void()> stop;
+    function<string(const string&)> command;
```

Comment:
Just a question, is this the function to execute a command?
It will also need a list of parameters and a callback, but we can modify this later.